### ISSUE TITLE
about zj_deviation and cvx toolbox

### ISSUE THREAD

**Issue by JR9124** [jjcao/pcv#1]
Hi, could you please upload the zj_deviation and cvx toolbox? Thank you!

**Comment by JanHurtado** [comment]
@JR9124 did you solved the kdtree crash?

**Comment by JanHurtado** [comment]
@JR9124 did you solve the kdtree crash ?

**Comment by JR9124** [comment]
> @JR9124 did you solve the kdtree crash ?

Yes, you can find kdtree toolbox in [https://github.com/jjcao/jjcao_code/tree/head/toolbox,=](url), you need to run kdtree_compile.m first, then there will be no crash.

I have read the code of pcv, however, it's not exactly the same as the paper stated, do you read the code? In addition, as randperm is used in the code, there is a little difference among different times, if you run the code for several times, you can find this. I don't know how to select the best one if I have run several times, do you know?
